### ISSUE TITLE
"What an interesting effect (882)"

### ISSUE THREAD

**Issue by mathematicalcoffee** [Soviet5lo/SlashTHEM#14]
I was an acid mage, applying my starting acid wand (hoping to kill myself "by breaking a wand" rather than "in eir own magical blast").

Anyhow, I survived the blast, and then got:

```
What an interesting effect (882)
 Program in disorder. Please file an issue at https://github.com/Soviet5lo/slashthem/issues.
Ugh!
```

It was this game: https://nethack.xd.cm/userdata/coffeebeetle/slashthem/dumplog/1434509043

**Comment by pirate486743186** [comment]
hmm, it seams this is minor bug.
They simply forgot coding this possibility.
The message is actually a warning.

**Comment by mathematicalcoffee** [comment]
Yes, it didn't seem to have an adverse effect or crash the game (unlike the other games of slashTHEM I've played....) - just thought I'd report it since the game told me to :)

**Comment by pirate486743186** [comment]
i meant, not to worry, it will not crash.
Save scum to protect your games against bugs and crashes.

**Comment by Soviet5lo** [comment]
Fixed in commit 71f4afe
